text/cognito-construct-library/working-backwards-readme.md

```diff
@@ -0,0 +1,300 @@
+# Amazon Cognito Construct Library
+
+Amazon Cognito provides authentication, authorization, and user management for your web and mobile apps. Your users can
+sign in directly with a user name and password, or through a third party such as Facebook, Amazon, Google or Apple.
+
+The two main components of Amazon Cognito are user pools and identity pools. User pools are user directories that
+provide sign-up and sign-in options for your app users. Identity pools enable you to grant your users access to other
+AWS services.
+
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+
+## User Pools
+
+User pools allows creating and managing your own directory of users that can sign up and sign in. They enable easy
+integration with social identity providers such as Facebook, Google, Amazon, Microsoft Active Directory, etc. through
+SAML.
+
+The CDK APIs provide ways to both create user pools, via the `UserPool` constructor, or import existing ones that were
+created outside the CDK App.
+
+User pools can be imported using the `UserPool.fromUserPoolAttributes()` API. They can be imported either using the
```

Comment:
Always start with the simplest getting started (Not sure it makes sense to start from the import case.).